tests/foreman/api/test_rhcloud_inventory.py

```diff
@@ -21,6 +21,7 @@
 import pytest
 from fauxfactory import gen_alphanumeric
 from fauxfactory import gen_string
+from nailgun import entities
```

Comment:
Same as in https://github.com/SatelliteQE/robottelo/pull/9510#discussion_r875714115, please use the Satellite object to access nailgun entities.